packages/battery_plus/battery_plus/android/src/main/java/dev/fluttercommunity/plus/battery/BatteryPlusPlugin.java

```diff
@@ -107,7 +110,7 @@ public void onListen(Object arguments, EventSink events) {
 
   private String getBatteryStatus() {
     int status;
-    if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP) {
+    if (android.os.Build.VERSION.SDK_INT >= VERSION_CODES.O) {
```

Comment:
BATTERY_PROPERTY_STATUS was added just in `O`, so there was an error here. However, this method was called from a method with annotation `TargetAPI(O)`, so this is why we haven't seen users reporting issues with this call.